### PR TITLE
[DEV-2342] Adding Coming Soon Section

### PR DESCRIPTION
**Pull request by MaxwellKendall**
**High level description:**
New `ComingSoonSection` Component with two different implementations: w/ header (Entire section is coming soon) w/o header (only part of section is coming soon.

**Technical details:**
Had to change the layout from using the bourbon flex `mixins`. For some reason they were not behaving in the same way that plain flex-box declarations do.

**JIRA Ticket:**
[DEV-2342](https://federal-spending-transparency.atlassian.net/browse/DEV-2342)

**Screenshot:**
image

**Mockup:**
image

**Commits:**
886ffd1, b669776, 0f1b9b8, a960a45 - Adding ComingSoonSection
3338661 -- Removing bourbon library flex-box mixins

Other commits are style related

The following are ALL required for the PR to be merged:
- [x] Code review
- [ ] Design review (if applicable)
- [ ] Verified cross-browser compatibility

### CODE REVIEW

File: src/js/components/awardv2/idv/ComingSoonSection.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    icon,
+    toolTip,
+    toolTipAlignLeft,
+    toolTipContent,
+    includeHeader,
+    children,
+    className
+}) => {
+    const renderChildren = () => (
+        children || (
```
Comment (ebdabbs) [inline src/js/components/awardv2/idv/ComingSoonSection.jsx]: Like that this component is very re-usable!

File: src/js/components/awardv2/idv/FundingSummary.jsx [hunk 2]
```diff
@@ -3,6 +3,9 @@ import PropTypes from 'prop-types';
 
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { formatMoneyWithPrecision } from '../../../helpers/moneyFormatter';
+import ComingSoonSection from "./ComingSoonSection";
+// import InfoToolTip from "./InfoTooltip";
```
Comment (ebdabbs) [inline src/js/components/awardv2/idv/FundingSummary.jsx]: Can this be removed for now?

File: src/js/components/awardv2/idv/ComingSoonSection.jsx (hunk 1 above)
Comment (MaxwellKendall) [reply src/js/components/awardv2/idv/ComingSoonSection.jsx]: 💪

File: src/_scss/selectors/_idv.scss [hunk 3]
```diff
@@ -6,3 +6,29 @@
     justify-content: space-between;
     color: #525252;
 }
+
+%coming-soon-idv {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: $color-gray-lightest;
+    padding: 2rem 2rem 4rem 2rem;
```
[the remark this answers is not in the document]
Comment (MaxwellKendall) [reply src/_scss/selectors/_idv.scss]: cool!

File: src/_scss/pages/awardV2/awardPage.scss [hunk 4]
```diff
@@ -66,25 +66,28 @@
...
-                @include display(flex);
-                @include flex-wrap(wrap);
-                @include justify-content(space-between);
-                @include align-items(flex-start);
+                display: flex;
+                flex-wrap: wrap;
                 padding: rem(40) 0 rem(60);
                 @include media($medium-screen) {
-                    @include flex-wrap(nowrap);
+                    flex-wrap: nowrap;
```
[the remark this answers is not in the document]
Comment (MaxwellKendall) [reply src/_scss/pages/awardV2/awardPage.scss]: Switched back!!

**Pull request merged**